### PR TITLE
Add feature enablement information

### PR DESCRIPTION
**Pull request by markdembo**
Added information on how to get WFP enabled

### CODE REVIEW

File: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ meta:
 
 # Configure Workers for Platforms
 
-This guide will instruct you on setting up Workers for Platforms. This guide assumes that you already have a Cloudflare account. If you do not have a Cloudflare account, sign up before continuing.
+This guide will instruct you on setting up Workers for Platforms. This guide assumes that you already have a Cloudflare account. If you do not have a Cloudflare account, sign up before continuing. Moreover, this feature is not enabled by default. Please contact your account manager, or reach out on Discord on more information on how to get Workers for Platforms enabled for your account.
```
Comment (tanushree-sharma) [inline content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md]:
```suggestion
This guide will instruct you on setting up Workers for Platforms. This guide assumes that you already have a Cloudflare account. If you do not have a Cloudflare account, sign up before continuing. Moreover, this feature is not enabled by default. Please contact your account manager, or reach out on the [Cloudflare Developer Discord](https://discord.com/invite/cloudflaredev) on more information about how to get Workers for Platforms enabled for your account.
```

**Comment by tanushree-sharma** [comment]
@deadlypants1973 can you take a look at this?

File: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ meta:
 
 # Configure Workers for Platforms
 
-This guide will instruct you on setting up Workers for Platforms. This guide assumes that you already have a Cloudflare account. If you do not have a Cloudflare account, sign up before continuing.
+This guide will instruct you on setting up Workers for Platforms. This guide assumes that you already have a Cloudflare account. If you do not have a Cloudflare account, sign up before continuing. Moreover, this feature is not enabled by default. Please contact your account manager, or reach out on Discord for more information on how to get Workers for Platforms enabled for your account.
```
Comment (deadlypants1973) [inline content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md:11]: talking with tanushree about updating this with exact instructions, do not merge

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

**Comment by deadlypants1973** [comment]
Closing this to open another PR with different formatting

**Comment by deadlypants1973** [comment]
https://github.com/cloudflare/cloudflare-docs/pull/6947

**Pull request closed without merging**